src/mozanalysis/metrics/__init__.py

```diff
@@ -282,16 +282,19 @@ class Metric:
 
 def agg_sum(select_expr):
     """Return a SQL fragment for the sum over the data, with 0-filled nulls."""
+    print("The use of mozanalysis.metrics.agg_sum() is deprecated")
```

Comment:
Any thoughts on raising Warnings instead of prints?

Reply:
I saw that mozanalysis has been using `print`s to log stuff so I figured to be consistent I do the same here. But maybe we should just use `logging` in the other places as well